pmaudit

```diff
@@ -490,6 +490,17 @@ def main():
         '-d', '--depsfile',
         metavar='FILE',
         help="save audit data in makefile format to FILE")
+    parser.add_argument(
+        '--shell',
+        metavar='SHELL_COMMAND',
+        help="Name of shell to run (default=SHELL else /bin/sh)")
+    parser.add_argument(
+        '--shellflags',
+        metavar='SHELL_FLAGS',
+        help="Space-separated set of flags to pass to shell (default=-c)")
```

Comment:
I've noted previously that I'm an unusually convention-driven programmer. My philosophy of programming is that nothing should be random and every decision establishes a precedent so one should create new code with great care and always observe existing code for existing conventions. For instance, the convention here is that help messages do not start with a capital letter.

Reply:
Done.